src/navbar/navbar-action-buttons.tsx

```diff
@@ -118,14 +120,23 @@ export const NavbarActionButtons = ({
         );
     };
 
-    const renderButtons = (isMobile = false) => {
+    const renderButtons = (
+        isMobile = false,
+        actionButtonList = actionButtons
```

Comment:
can be explicit about the arguments now

```suggestion
        isMobile: boolean,
        actionButtonList: NavbarButtonProps[]
```